src/type-translator.ts

```diff
@@ -503,13 +504,15 @@ export class TypeTranslator {
   /** @return true if sym should always have type {?}. */
   isBlackListed(symbol: ts.Symbol): boolean {
     if (this.pathBlackList === undefined) return false;
-    const pathBlackList = this.pathBlackList;
     if (symbol.declarations === undefined) {
       this.warn('symbol has no declarations');
       return true;
     }
+    // Normalize paths to not break checks on Windows.
+    const pathBlackList = new Set<string>(
```

Comment:
Should I keep the auto assignment of `pathBlackList` and just overwrite the property like this in the constructor?

```typescript
this.pathBlackList = new Set<string>(
        Array.from(this.pathBlackList.values()).map(path => nodePath.normalize(path)));
```